modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts

```diff
@@ -109,25 +119,30 @@ export default class I3SNodePagesTiles {
       lodSelection,
       obb: node.obb,
       contentUrl,
+      // @ts-ignore
```

Comment:
Please use `@ts-expect-error`, so that we can remove these when types are fixes.

Reply:
Re-done types here